src/pages/Payouts/PayoutList/index.tsx

```diff
@@ -139,126 +52,44 @@ export const PayoutListInner = (props: PayoutListProps) => {
-            );
-
-            const batchIndex = validator
-              ? validators.indexOf(validator)
-              : pool
-              ? bondedPools.indexOf(pool)
-              : 0;
-
-            return (
-              <motion.div
-                className={`item ${listFormat === 'row' ? 'row' : 'col'}`}
-                key={`nomination_${index}`}
-                variants={{
-                  hidden: {
-                    y: 15,
-                    opacity: 0,
-                  },
-                  show: {
-                    y: 0,
-                    opacity: 1,
-                  },
-                }}
-              >
-                <ItemWrapper>
-                  <div className="inner">
-                    <div className="row">
-                      <div>
-                        <div>
-                          <h4 className={`${labelClass}`}>
-                            {p.event_id === 'Slashed' ? '-' : '+'}
-                            {planckToUnit(p.amount, units)} {network.unit}
-                          </h4>
-                        </div>
-                        <div>
-                          <h5 className={`${labelClass}`}>{label}</h5>
-                        </div>
-                      </div>
-                    </div>
-                    <div className="row">
-                      <div>
-                        <div>
-                          {label === t('payouts.payout') && (
-                            <>
-                              {batchIndex > 0 ? (
-                                <Identity
-                                  meta={meta}
-                                  address={p.validator_stash}
-                                  batchIndex={batchIndex}
-                                  batchKey={batchKey}
-                                />
-                              ) : (
-                                <div>{clipAddress(p.validator_stash)}</div>
-                              )}
-                            </>
-                          )}
-                          {label === t('payouts.pool_claim') && (
-                            <>
-                              {pool ? (
-                                <PoolIdentity
-                                  batchKey={batchKey}
-                                  batchIndex={batchIndex}
-                                  pool={pool}
-                                />
-                              ) : (
-                                <h4>
-                                  {t('payouts.from_pool')} {p.pool_id}
-                                </h4>
-                              )}
-                            </>
-                          )}
-                          {label === t('payouts.slashed') && (
-                            <h4>{t('payouts.deducted_from_bond')}</h4>
-                          )}
-                        </div>
+          {payouts
```

Comment:
This big code reduction is an aftermath of switching from the complex, subscan-based data to a simple payout-per-era structure.